test/powershell/Language/Classes/scripting.enums.tests.ps1

```diff
@@ -3,6 +3,11 @@
 #
 
 Describe 'enums' -Tags "CI" {
+    Context 'Enum definition' {
+        It "Enum values definition have space-insensitive syntax" {
+            { enum E0 {e=0} } | Should Not Throw

```

Comment:
You can use `[scriptblock]::Create`, like in https://github.com/PowerShell/PowerShell/blob/ca0be9ad6103c1ef0b48b52ae0a85666f7bf2332/test/powershell/Language/Classes/Scripting.Classes.BasicParsing.Tests.ps1#L665